packages/plugin-dom/DomRenderer.ts

```diff
@@ -75,7 +75,10 @@ export class DomRenderer extends Renderer<
         for (const renderingFunction of this.renderingFunctions) {
             const renderingResult = renderingFunction({ ...context });
             if (renderingResult) {
-                return renderingResult;
+                context = renderingResult[0];
```

Comment:
This is why I call it "dirty make-it-work" ;-)
How should we handle this kind of situation? We need to be able to render a wrapper, then let the node render itself within said wrapper.
Also there is an issue of priorities. By chance right now it works, but it's only by chance: List has to take control whenever a ListNode's children have to be rendered. But if another renderingFunction renders the node, List will never be called. List _could_ handle the rendering of all its children, like it was done in the non-agnostic renderer... but then they need to be able to call `render` on each of said children.